.tekton/tasks/task-publish-code-engine-deployable-mapping.yml

```diff
@@ -0,0 +1,141 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
We might want to consider publishing this to the tekton-catalog repo. It's useful to have for everyone.